arch/arc/soc/em11d/soc.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 Synopsys, Inc. All rights reserved.
+ * Copyright (c) 2017 Synopsys, Inc. All rights reserved.
```

Comment:
> Unless I am mistaken, if you want to update the date keep the very first one and add the new one besides: s/2016/2016-2017

I believe per the copyright law, last date of work is important: 50 years after that, it's in public domain. Starting date is just a trivia for curious (which I personally appreciate).